packages/support/resources/css/components/input/index.css

```diff
@@ -36,3 +36,9 @@ input.fi-input {
         @apply text-justify;
     }
 }
+
+input[type='range'].fi-input {
+    margin-inline: auto;
+    width: 98%;
```

Comment:
Instead of a percentage width which will change with screen size, maybe add a fixed `margin-inline` of `1rem` instead?

Maybe `@apply mx-4 appearance-auto;`? How does that look?